registers/dump/main.go

```diff
@@ -1,22 +1,69 @@
 package main
 
 import (
+	"context"
+	"encoding/json"
 	"flag"
+	"fmt"
 	"log"
 
 	"github.com/benlaurie/gds-registers/register"
+	"github.com/google/trillian"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
 )
 
 var (
-	regName = flag.String("register", "register", "name of register (e.g. 'country')")
+	regName     = flag.String("register", "register", "name of register (e.g. 'country')")
+	trillianLog = flag.String("trillian_log", "localhost:8090", "address of the Trillian Log RPC server.")
+	logID       = flag.Int64("log_id", 0, "Trillian LogID to populate.")
```

Comment:
The Go style rules say you have to use `logId` not `logID`.

Reply:
No - the opposite, `logID` is correct :) ([see here](https://github.com/golang/go/wiki/CodeReviewComments#initialisms))